### DISCUSSION TITLE
Vendor neutral documentation

### DISCUSSION THREAD

**Discussion by Lunarequest** [cockpit-project/cockpit#20207] [General]
while the cockpit project has a decent amount of vendor neutral documentation at https://cockpit-project.org/documentation.html there are many links that point to redhat documentation. On top of this all of the help links inside cockpit point to redhat.

If we could move all the documentation to https://cockpit-project.org at SUSE we would be able to leave the documentation as it is. However as it points to redhat documentation we have to make our own documentation duplicating work. A single vendor neutral documentation would allow us to unify our documentation efforts and produce a better higher quality document

**Comment by martinpitt** [comment]
We can't "move" the access.rh.com docs -- these are the RHEL product documentation, and our team isnt' even writing them, and we can't just maintain a "fork" of them for both legal and also capacity reasons.

We actually keep the RHEL docs links in the Debian/Ubuntu/arch packages as well -- they are publicly accessible, and we found that better than disabling them entirely.

However, I understand if you don't want to link to them in openSUSE -- so how about we provide a build-time knob to hide them?

Most of them are in manifest.json, so we can either remove them at `make install` time (with an appropriate `./configure` option) or handle this in the shell to just ignore them. The rest is some direct links in the pages:

> pkg/networkmanager/bond.jsx:                                          href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_systems_using_the_rhel_9_web_console/configuring-network-bonds-using-the-web-console_system-management-using-the-rhel-9-web-console">
> pkg/systemd/hwinfo.jsx:                                        <a href="https://access.redhat.com/security/vulnerabilities/L1TF" target="_blank" rel="noopener noreferrer">
> pkg/systemd/overview-cards/cryptoPolicies.jsx:                    href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening">
> pkg/systemd/overview-cards/cryptoPolicies.jsx:                        href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/using-the-system-wide-cryptographic-policies_security-hardening">
> pkg/systemd/overview-cards/tuned-dialog.jsx:                        href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/managing_systems_using_the_rhel_8_web_console/index#optimizing-the-system-performance-using-the-web-console_system-management-using-the-RHEL-8-web-console">

We can certainly find a way to conditionalize these.
